src/fiber/schema/gossip.mol

```diff
@@ -0,0 +1,168 @@
+// The `flags` is used to specify an array of bitfields. Bits have the following meaning:
+// 0 	Sender wants ChannelAnnouncement
+// 1 	Sender wants ChannelUpdate for node 1
+// 2 	Sender wants ChannelUpdate for node 2
+// 3 	Sender wants NodeAnnouncement for node 1
+// 4 	Sender wants NodeAnnouncement for node 2
+// The bit field order of these messages is the same to the union id order of cursors.
+struct BroadcastMessageQuery {
+    channel_outpoint: OutPoint,
+    flags: byte,
+}
+
+vector BroadcastMessageQueries <BroadcastMessageQuery>;
+
+// A NodeAnnouncement is a broadcast message to the network for the node information.
+// An update to the node information can be broadcasted with a NodeAnnouncement with larger timestamp. 
+table NodeAnnouncement {
+    // Signature to this message.
+    signature: EcdsaSignature,
+    // Tentatively using 64 bits for features. May change the type later while developing.
+    // https://github.com/lightningdevkit/rust-lightning/blob/8c1b3d1263f6b9247c0c819039ef2027dc4d4588/lightning/src/ln/features.rs#L448-L457
+    // rust-lightning uses a Vec<u8> here.
+    features: Uint64,
+    // Timestamp to the node announcement update, later update should have larger timestamp.
+    timestamp: Uint64,
+    node_id: Pubkey,
+    // Must be a valid utf-8 string of length maximal length 32 bytes.
+    alias: Byte32,
+    // All the reachable addresses.
+    address: BytesVec,
+    // Chain hash
+    chain_hash: Byte32,
+    // Minimal auto accept ckb amount
+    auto_accept_min_ckb_funding_amount: Uint64,
+    // UDT configs
+    udt_cfg_infos: UdtCfgInfos,
+}
+
+// A ChannelAnnouncement is an announcement of a channel to the network.
+// This message must be immutable for any specific channel.
+table ChannelAnnouncement {
+    node1_signature: EcdsaSignature,
+    node2_signature: EcdsaSignature,
+    // Signature signed by the funding transaction output public key.
+    ckb_signature: SchnorrSignature,
+    // Tentatively using 64 bits for features. May change the type later while developing.
+    // https://github.com/lightningdevkit/rust-lightning/blob/8c1b3d1263f6b9247c0c819039ef2027dc4d4588/lightning/src/ln/msgs.rs#L1200-L1200
+    // rust-lightning uses a Vec<u8> here.
+    features: Uint64,
+    chain_hash: Byte32,
+    channel_outpoint: OutPoint,
+    node1_id: Pubkey,
+    node2_id: Pubkey,
+    // The aggregated public key of the funding transaction output.
+    ckb_key: SchnorrXOnlyPubkey,
+    capacity: Uint128,
+    udt_type_script: ScriptOpt,
+}
+
+// A ChannelUpdate is a update to the public channel. Newer channel information may be updated with
+// a ChannelUpdate with a larger timestamp.
+table ChannelUpdate {
+    // Signature of the node that wants to update the channel information.
+    signature: EcdsaSignature,
+    chain_hash: Byte32,
+    channel_outpoint: OutPoint,
+    timestamp: Uint64,
+    message_flags: Uint32,
+    channel_flags: Uint32,
```

Comment:
If we update `enable` from channel update rpc from `true` to `false`, we need to update this `channel_flags`?

Reply:
This should be done in a separated PR.